app/views/layouts/_nav.haml

```diff
@@ -0,0 +1,30 @@
+%nav.navbar.navbar-default.nav-custom
+  .container-fluid
+    .navbar-header
+      %button.navbar-toggle.collapsed{ type: 'button',
+       data: { toggle: 'collapse', target: '#bs-example-navbar-collapse-1' },
+        aria: { expanded: 'false' } }
+
```

Comment:
Ah, I wasn't being very observant when I made that comment. As someone less experienced with haml and well, code in general, it made it more readable to me. I see what you mean now though. I'll make sure to keep it consistent with the pros. 👍